lib/desktop/taskbar/taskbar_item.dart

```diff
@@ -59,13 +60,18 @@ class _TaskbarItemState extends State<TaskbarItem>
 
   @override
   Widget build(BuildContext context) {
+    //Selected App
+    final _app = applications
+        .firstWhere((element) => element.packageName == widget.packageName);
+
+    if (!_app.canBeOpened) {
+      return SizedBox.shrink();
```

Comment:
dont know if its the best idea to have such a thing? it would break some logic such as reorder, imo just disable pointer events on the item and make it translucent, like 20% opacity

Reply:
It wouldn't break anything cuz the widget is not removed from the tree, it is simply a 0x0 box. I don't see why it would break reorder

Reply:
it's because it still takes a slot in the taskbar, plus imo its just better to have a disabled button meaning that the app exists but its just not supported on web, so better ux